packages/express-did-auth/src/index.ts

```diff
@@ -40,7 +40,7 @@ export default function setupAppFactory (config: ExpressDidAuthConfig) {
 
     if (config.useCookies) {
       app.use(cookieParser())
-      app.use(csrf({ cookie: { httpOnly: true } }))
+      app.use(csrf({ cookie: { httpOnly: false, secure: true } }))
```

Comment:
Maybe I wasn't that clear in my previous comment about this, i'm sorry. 

I was referring to the header on 45. It should be a cookie instead, with httpOnly as false. The cookie on 43 was good with httpOnly as true